tests/sentry/tasks/integrations/github/test_pr_comment.py

```diff
@@ -0,0 +1,180 @@
+from sentry.models import Commit, GroupOwner, GroupOwnerType, PullRequest
+from sentry.tasks.integrations.github import pr_comment
```

Comment:
Looks good to me. Thanks for thorough testing.

Should we rename this since the pr_to_issue_query doesn't have anything to do with comment. I find it confusing when I see:
``
pr_comment.pr_to_issue_query()
``
It sounds as if getting a single comment it returns the related issue which is not.

Reply:
the init file (empty or not) is needed in order for python to treat the directory as a package. We can then import the module in other locations.

This module will eventually hold all the logic for making the github comment. For right now, it's just this helper.

Reply:
eventually it'll look something like 
```
def run_task():
  results = pr_to_issue_query()
  for repo_id, pr_key, org_id, issue_list in results :
    # rest of logic
```